jwst/rscd/rscd_step.py

```diff
@@ -8,47 +8,65 @@
 
 class RscdStep(Step):
     """
-    RscdStep: Performs an RSCD correction to MIRI data.
-    Baseline version flags the first N groups as 'DO_NOT_USE' in
-    the 2nd and later integrations in a copy of the input
-    science data model.
-    Enhanced version is not ready nor enabled.
+    Flags the first N groups of MIRI data to 'DO_NOT_USE' in the 2nd and later integrations.
+
+    Input data is expected to be a ramp file (RampModel). The number of groups, N,  to
+    set the GROUPDQ flag to 'DO_NOT_USE' is read in from the RSCD reference file. The RSCD
+    reference file contains the number of groups to set to 'DO_NOT_USE' based on readout mode and
+    subarray size. The step checks that the total number of groups in an integration is
+    greater than N+3 before flagging the GROUPDQ array. If the number of groups is less than
+    N+3 then no flagging is performed, because doing so would leave too few groups to work
+    with in later steps.
     """
 
     class_alias = "rscd"
 
     # allow switching between baseline and enhanced algorithms
     spec = """
-        type = option('baseline','enhanced',default = 'baseline') # Type of correction
-        """ # noqa: E501
+        """  # noqa: E501
 
-    #  TBD - only do this for the 2nd+ integrations
+    #  only do this for the 2nd+ integrations
     #  do nothing for single integration exposures
```

Comment:
suggest removing this comment entirely, as its information is inside the docstring now